test/SlowTests/Authentication/AuthenticationBasicTests.cs

```diff
@@ -401,7 +401,8 @@ public void AllAdminAuthorizationStatusHaveCorrectRoutes()
             {
                 "/monitoring/snmp/oids",
                 "/monitoring/snmp",
-                "/monitoring/snmp/bulk"
+                "/monitoring/snmp/bulk",
+                "/topology/modify"
```

Comment:
why is that needed here? looks wired to me it is the only EP of that type here..

Reply:
Removed